docs/changelog/next_release/59.breaking.rst

```diff
@@ -0,0 +1,3 @@
+Rename attribute ``ColumnHWM.column`` to ``ColumnHWM.source``. This is not requires anymore, and should contain table/collection/topic name instead of column.
```

Comment:
```suggestion
Rename attribute ``ColumnHWM.column`` to ``ColumnHWM.source``. This is not required anymore, and should contain table/collection/topic name instead of column.
```